doc/release-notes/9.0.md

```diff
@@ -0,0 +1,263 @@
+- [Major changes and features](#major-changes-and-features)
+- [Major issues](#major-issues)
+- [Bugs addressed in the release](#bugs-addressed)
+
+## Announcements
+
+1. Releases that receive maintenance updates post release 9 is 8 
+([reference](https://www.gluster.org/release-schedule/))
+
+2. Release 9 will receive maintenance updates around the 30th of every month
+for the first 3 months post release (i.e Jan'21, Feb'21, Mar'21). Post the
+initial 3 months, it will receive maintenance updates every 2 months till EOL.
+
+
+
+## Major changes and features
+
+### Highlights
+
+- Added support for:
+
+  - IO Uring in Gluster
+  - 5K volumes 
+
+- Several fixes addressing
+
+  - Code optimization and improvements, bugs and other identified fixes from earlier 8.x releases
+
+### Features
+
+- Added io_uring support for Gluster [#1398](https://github.com/gluster/glusterfs/issues/1398)
+- Added Support for 5K volumes [#1613](https://github.com/gluster/glusterfs/issues/1613)
+- Enabled granular-entry-heal by default [#1483](https://github.com/gluster/glusterfs/issues/1483)
+- Optimizations for rename dir heal [#1211](https://github.com/gluster/glusterfs/issues/1211)
+- Added support for monitoring the epoll/rpc layer  [#1466](https://github.com/gluster/glusterfs/issues/1466) 
+- Brick mux: Added support to spawn a thread per process basis instead of spawning a per brick [#1482](https://github.com/gluster/glusterfs/issues/1482)
+
+## Major issues
+
+**None**
+
+
+## Bugs addressed
+
+Bugs addressed since release-8 are listed below.
+
+
+- [#718](https://github.com/gluster/glusterfs/issues/718)  _store_global_opts(), _storeslaves() , _storeopts() should no...
+- [#280](https://github.com/gluster/glusterfs/issues/280)  Use internal error codes instead of UNIX errnos
+- [#1855](https://github.com/gluster/glusterfs/issues/1855) Makefile: failed to compile without git repository
+- [#1849](https://github.com/gluster/glusterfs/issues/1849) geo-rep: The newly setup geo-rep session goes faulty with syn...
+- [#1836](https://github.com/gluster/glusterfs/issues/1836) posix: Update ret value in posix_get_gfid2path if GF_MALLOC f...
+- [#1810](https://github.com/gluster/glusterfs/issues/1810) Implement option to generate core dump at will without killin...
+- [#1796](https://github.com/gluster/glusterfs/issues/1796) afr: call afr_is_lock_mode_mandatory only while xdata is valid
+- [#1794](https://github.com/gluster/glusterfs/issues/1794) posix: A brick process is getting crashed at the time of grap...
+- [#1782](https://github.com/gluster/glusterfs/issues/1782) Rebalance is reporting status twice upon stopping, resulting ...
+- [#1778](https://github.com/gluster/glusterfs/issues/1778) volume set: failed: ganesha.enable is already 'off'.
+- [#1775](https://github.com/gluster/glusterfs/issues/1775) core: lru_size showing -1 with zero inodes in the list in the...
+- [#1772](https://github.com/gluster/glusterfs/issues/1772) build: add LTO as a configure option
```

Comment:
this one is worth highlighting under Features, IMHO.

Reply:
Done